fv3core/decorators.py

```diff
@@ -337,8 +337,15 @@ class FixedOriginStencil:
     and using the normalized origins."""
 
     def __init__(self, func, origin, domain, **kwargs):
-        self.normalized_origin = gtscript.gt_definitions.normalize_origin_mapping(
-            origin
+        self.normalized_origin = (
```

Comment:
Should we assert that `origin` or `domain` be not `None` in `__init__` since they cannot be passed to `__call__`, or does the `gt4py` code handle that?

Reply:
so for the normalized origin it is handled by creating an empty dict, this is as if you would call a gt4py stencil without an origin. And for the domain can be None for the same reason

Reply:
Sweet!